app/services/log_event_searcher.rb

```diff
@@ -5,7 +5,8 @@ class LogEventSearcher
   ALLOWED_EVENTS = ["account.create", "account.update",
                     "account_user.create", "account_user.delete",
                     "user.create", "user.suspended", "user.unsuspended",
-                     "account.suspended", "account.unsuspended"].freeze
+                    "account.suspended", "account.unsuspended", 
+                    "statement.create"].freeze
```

Comment:
Whenever the next thing goes in, can we move `].freeze` onto its own line. That should make the diffs a little cleaner (as long as we always put a comma after the last item in the list).